File: infra/cdk/Stack.java
import software.amazon.awscdk.services.s3.Bucket;
import software.amazon.awscdk.services.s3.BucketProps;

final class Stack {
  void build() {
    new Bucket(
        this,
        "JavaAssets",
        BucketProps.builder().bucketName("repello-cdk-java-assets").build());
  }
}

File: infra/cdk/Stack.cs
using Amazon.CDK.AWS.S3;

internal sealed class Stack
{
    public void Build()
    {
        new Bucket(this, "CsharpAssets", new BucketProps
        {
            BucketName = "repello-cdk-csharp-assets"
        });
    }
}
